js/src/repo.js

```diff
@@ -6,8 +6,18 @@
 const chai = require('chai')
 const dirtyChai = require('dirty-chai')
 const expect = chai.expect
+const Big = require('big.js')
 chai.use(dirtyChai)
 
+const isNumber = (n) => {
```

Comment:
This is actually checking if `n` is a BigInt, not `Number`, which is a built-in type in JavaScript